package Cap16.exceptions2.runtime.test;

public class RuntimeExceptionTest04 {
    public static void main(String[] args) {

    }
}
